configs/model/synth_model.yaml

```
@@ -0,0 +1,56 @@
+
+# The model sites used to register the keypoints.
+KEYPOINT_MODEL_PAIRS:
+  part_0: base
+
+# The initial offsets for each keypoint in meters.
+KEYPOINT_INITIAL_OFFSETS:
+  part_0: 0 0 0.01
+
+TRUNK_OPTIMIZATION_KEYPOINTS:
+  -part_0
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)